### PR TITLE
Update docstrings

### PR DESCRIPTION
**Pull request by alexdanjou**
* **Summary of changes**

* **Please check if the PR fulfills these requirements**

- [ ] Closes #xxxx (Replace xxxx with the Github issue number)
- [ ] Tests added and passing
- [x] Documentation and tutorials updated/added
- [x] Added an entry to the `CHANGELOG.md` file
- [ ] Added any new requirements to `requirements.txt`

### CODE REVIEW

File: openghg_inversions/hbmcmc/inversion_pymc.py [hunk 1]
```diff
@@ -372,91 +390,86 @@ def inferpymc(
...
+    rerun_file: Optional[xr.Dataset]=None,
     use_bc: bool = False,
     min_error: Union[float, np.ndarray] = 0.0,
 ) -> xr.Dataset:
     """
     Takes the output from inferpymc function, along with some other input
-    information, and places it all in a netcdf output. This function also
-    calculates the mean posterior emissions for the countries in the
-    inversion domain and saves it to netcdf.
+    information, calculates staitics on them and places it all in a dataset.
```
Comment (brendan-m-murphy) [inline openghg_inversions/hbmcmc/inversion_pymc.py:437]:
```suggestion
    information, calculates statistics on them and places it all in a dataset. 
```

File: openghg_inversions/hbmcmc/inversion_pymc.py [hunk 2]
```diff
@@ -372,91 +390,86 @@ def inferpymc(
...
+      error:
         Measurement error vector, containg a value for each element of Y.
-      Ytrace (array):
+      Ytrace:
         Trace of modelled y values calculated from mcmc outputs and H matrices
-      YBCtrace (array):
-        Trace of modelled boundary condition values calculated from mcmc outputs and Hbc matrices
-      step1 (str):
+      OFFSETtrace:
+        XXXX.
```
Comment (brendan-m-murphy) [inline openghg_inversions/hbmcmc/inversion_pymc.py:467]:
```suggestion
        Trace from offsets (if used).
```

File: openghg_inversions/hbmcmc/inversion_pymc.py [hunk 3]
```diff
@@ -466,64 +479,75 @@ def inferpymc_postprocessouts(
         e.g. N(1,1**2) would be: xprior={pdf:"normal", "mu":1, "sd":1}.
```
Comment (brendan-m-murphy) [inline openghg_inversions/hbmcmc/inversion_pymc.py:479]:
```suggestion
        e.g. N(1,1**2) would be: xprior={pdf:"normal", "mu":1, "sigma":1}.
```

File: openghg_inversions/hbmcmc/inversion_pymc.py [hunk 4]
```diff
@@ -466,64 +479,75 @@ def inferpymc_postprocessouts(
...
+      bcprior:
+        Same as xrpior but for boundary conditions.
+      YBCtrace:
+        Trace of modelled boundary condition values calculated from mcmc outputs and Hbc matrices
+      bcouts:
+        MCMC chain for boundary condition scaling factors.
+      Hbc:
+        Same as Hx but for boundary conditions
+      obs_repeatability: 
+        XXXX
```
Comment (brendan-m-murphy) [inline openghg_inversions/hbmcmc/inversion_pymc.py:532]:
```suggestion
        Instrument error
```

File: openghg_inversions/hbmcmc/inversion_pymc.py [hunk 5]
```diff
@@ -466,64 +479,75 @@ def inferpymc_postprocessouts(
...
+      obs_variability: 
+        XXXX
```
Comment (brendan-m-murphy) [inline openghg_inversions/hbmcmc/inversion_pymc.py:534]:
```suggestion
        Error from resampling observations
```

**Review by brendan-m-murphy — COMMENTED** [review]
Looks good, just commit the suggestions if you agree with them

**Comment by alexdanjou** [comment]
Thanks for the correction and comment Brendan! I updated the header of hbmcmc.py, hope it is ok now.

**Review by brendan-m-murphy — APPROVED** [review]
Looks good, thanks! I will look into fixing the failing tests with openghg devel. We can merge this now if we like, or wait until the tests are working (but it has nothing to do with your changes)

**Pull request merged**